components/search_results/search_results.tsx

```diff
@@ -49,13 +55,15 @@ const renderView = (props: Record<string, unknown>): JSX.Element => (
 const renderThumbHorizontal = (props: Record<string, unknown>): JSX.Element => (
     <div
         {...props}
+        style={scrollbarStyles}
         className='scrollbar--horizontal'
     />
 );
 
 const renderThumbVertical = (props: Record<string, unknown>): JSX.Element => (
     <div
         {...props}
+        style={scrollbarStyles}
         className='scrollbar--vertical'
```

Comment:
```suggestion
        className='scrollbar--vertical scrollbar--position'
```
we can define this class where have defined `scrollbar--vertical`